### PR TITLE
Update to Occlum 0.29.7

### PR DESCRIPTION
**Pull request by mythi**
Fixes: #126

### CODE REVIEW

File: src/runtime-boot/init/src/main.rs [hunk 1]
```diff
@@ -44,17 +41,22 @@ fn main() -> Result<(), Box<dyn Error>> {
     // Mount the image
     const SYS_MOUNT_FS: i64 = 363;
 
+    // Example envs. must end with null
+    let env1 = CString::new("TEST=1234").unwrap();
```
Comment (piotrpalcz) [inline src/runtime-boot/init/src/main.rs:45]: Why use example env set to that particular value?

Comment (mythi) [reply src/runtime-boot/init/src/main.rs:45]: I kept it here to see it works.

**Comment by mythi** [comment]
blocked by https://github.com/occlum/occlum/issues/1343

**Comment by Xynnn007** [comment]
Hi. I have test with the change https://github.com/confidential-containers/guest-components/pull/315

also, after the refactoring of guest-components dep, it is need to regenerate a `Cargo.lock` file.

**Comment by Xynnn007** [comment]
merged. please try this rev [fe8fb1a](https://github.com/confidential-containers/guest-components/commit/fe8fb1a2ff19d8568182b3dce113ddb2a72015ab)

**Comment by mythi** [comment]
Thanks, I'll try asap

**Comment by mythi** [comment]
> merged. please try this rev [fe8fb1a](https://github.com/confidential-containers/guest-components/commit/fe8fb1a2ff19d8568182b3dce113ddb2a72015ab)

@Xynnn007 all green, thanks!

**Pull request merged**